Jenkinsfile

```diff
@@ -22,6 +22,8 @@ pipeline {
                 // Checkout hootenanny
                 git url: 'https://github.com/ngageoint/hootenanny', branch: 'develop'
                 sh "git submodule init; git submodule update; cd hoot-ui; git checkout ${env.GIT_COMMIT}"
+                // Remove any screenshots from previous builds
+                sh "rm -rf ./test-files/ui/screenshot_*.png"
```

Comment:
This shouldn't strictly be necessary (capybara should remove screenshots on it's next run) but it won't hurt.